source/PlayerInfo.h

```diff
@@ -458,7 +461,8 @@ class PlayerInfo {
 	std::pair<const System *, Point> interstellarEscortDestination;
 	// Currently selected coloring, in the map panel (defaults to reputation):
 	int mapColoring = -6;
-	int mapZoom = 0;
+	float mapZoom = 0;
+	bool isStarry;
```

Comment:
```suggestion
	bool isStarry = false;
```
or whatever default value you want